.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "core/src/test/resources/tispark-test"]
+	path = core/src/test/resources/tispark-test
+	url = https://github.com/Novemser/tispark_test_data.git
```

Comment:
Test data are managed in a separate repository.

Reply:
Brilliant! So we are not concerning Chinese characters any more, are we?

Reply:
No, we don't need to consider Chinese characters ordering in `blob` representation any more. :laughing: